Cargo.toml

```diff
@@ -12,7 +12,7 @@ lto = "fat"
 codegen-units = 1
 
 [workspace.package]
-version = "3.1.2"
+version = "3.1.3"
```

Comment:
This likely needs to be updated to a different number, since we're going with the `3.2.x` scheme I believe.